docker/package/wizard_structure.py

```diff
@@ -196,7 +196,7 @@ def show_systemd_service(service_name):
 
 
 def find_systemd_env_files(show_systemd_output):
-    return re.findall(b"EnvironmentFiles=(.*) ", show_systemd_output)
+    return re.findall(b"EnvironmentFiles?=(.*) ", show_systemd_output)
```

Comment:
This seems to fix the problem here, however when I tested the `tezos-setup` I ended up with a failure either way, the output ends with:
```
Waiting for the node to bootstrap...
Fatal error:
  Unix.Unix_error(Unix.EACCES, "mkdir", "/home/fresh/.tezos-client")
Error in Tezos Setup Wizard, exiting.
The error has been logged to /home/fresh/tezos_setup.log
```
and the error logged is:
```
Command '['sudo', '-u', 'tezos', 'TEZOS_CLIENT_UNSAFE_DISABLE_DISCLAIMER=YES', 'octez-client', '--endpoint', 'http://127.0.0.1:8732', 'bootstrapped']' returned non-zero exit status 1.
```

Reply:
The issue might be in `octez-node-wrapper` :thinking: 

That also seems to be looking for the `EnvironmentFiles` (with the `s`) and probably needs to be updated as well

Reply:
You are right, but it's not the source of issue: it's failure of `octez-client`. But it's rather confusing, I had same issue and than it resolved by its own. Despite being called with `sudo -u tezos` it tries to create client dir at your user home dir, which is misbehavior. I need to investigate further.

Reply:
```
vagrant@ubuntu-bionic:~$ systemctl show -p EnvironmentFiles tezos-node-limanet.service 
EnvironmentFile=/etc/default/tezos-node-limanet (ignore_errors=no)
```

Reply:
🤔

Reply:
Indeed the `show` command seems to work correctly with the `s` on `jammy` too :ok_hand: 

Closing this thread as there is now a separate commit handling the other issue I mentioned above too